crates/sui-sdk/src/apis.rs

```diff
@@ -47,7 +48,23 @@ impl ReadApi {
     pub(crate) fn new(api: Arc<RpcClient>) -> Self {
         Self { api }
     }
-
+    /// Return a paginated response containing the owned objects for this Sui address, or an error upon failure.
+    ///
+    /// # Examples
+    ///
+    /// ```rust,no_run
+    /// use sui_sdk::SuiClientBuilder;
+    /// use sui_types::base_types::SuiAddress;
+    /// use std::str::FromStr;
+    ///
+    /// #[tokio::main]
+    /// async fn main() -> Result<(), anyhow::Error> {
+    ///     let sui = SuiClientBuilder::default().build_localnet().await?; // local Sui network
```

Comment:
I vaguely recall that it's possible to hide set-up code (`use` statements, the `main` function boilerplate, etc) to limit noise in the doc comments -- have a look at the output you get from `cargo doc` but that may be something worth considering for these.

Reply:
We can do that for sure, but I think it might be useful to have them nevertheless. Tokio seems to have most examples like that: https://docs.rs/tokio/1.29.1/tokio/net/fn.lookup_host.html.

Reply:
Fair, your call!  I think tokio is a little bit of a special case example though because it "takes over" the implementation of main, so it's relevant to include that in their docs (but you could also make an argument that in our case, it's worth knowing that we use tokio).